src/drive/web/modules/drive/AddMenu/AddMenu.jsx

```diff
@@ -42,7 +42,7 @@ export const ActionMenuContent = ({
         <AddEncryptedFolderItem />
       )}
       {!isPublic && !isEncryptedFolder && <CreateNoteItem />}
-      {canUpload && isOnlyOfficeEnabled() && !isEncryptedFolder && (
+      {canUpload && isOfficeEnabled() && !isEncryptedFolder && (
```

Comment:
not sure about this renaming. Everywhere we still use OnlyOffice. The links that this condition display are related to OO and so on